compiler/light-classes/src/org/jetbrains/kotlin/asJava/LightClassUtil.kt

```diff
@@ -185,6 +169,15 @@ object LightClassUtil {
         return null
     }
 
+    private fun getWrappingClasses(declaration: KtDeclaration): List<PsiClass> {
```

Comment:
You can simplify this code by changing the return type to `Sequence<PsiClass>` and using `generateSequence()`